packages/form-core/test-suites/FormatMixin.suite.js

```diff
@@ -300,6 +307,8 @@ export function runFormatMixinSuite(customConfig) {
 
         el.formattedValue = 'raw';
         expect(parserSpy.called).to.equal(true);
+        el.dispatchEvent(new CustomEvent('user-input-changed'));
+        expect(preprocessorSpy.called).to.equal(true);
```

Comment:
Maybe it would be nice to test the order of function calls here as well, since - especially for the preprocessor - it's vital that it runs before the rest